Actions/DetermineProjectsToBuild/DetermineProjectsToBuild.Action.ps1

```diff
@@ -21,37 +21,41 @@ if ($ENV:GITHUB_EVENT_NAME -eq 'pull_request') {
     $targetBranch = $env:GITHUB_BASE_REF
 }
 
+#region Action: Determine projects to build
 Write-Host "$($ENV:GITHUB_EVENT_NAME) on $targetBranch"
 $buildAllProjects, $publishSkippedProjects = Get-BuildAllProjectsBasedOnEventAndSettings -ghEventName $ENV:GITHUB_EVENT_NAME -settings $settings
 
+$modifiedFiles = @()
 $baselineWorkflowRunId = 0 #default to 0, which means no baseline workflow run ID is set
 $baselineWorkflowSHA = ''
 if(-not $buildAllProjects) {
     Write-Host "::group::Determine Baseline Workflow ID"
     $baselineWorkflowRunId,$baselineWorkflowSHA = FindLatestSuccessfulCICDRun -repository $env:GITHUB_REPOSITORY -branch $targetBranch -token $token -retention $settings.incrementalBuilds.retentionDays
     Write-Host "::endgroup::"
-}
 
-#region Action: Determine projects to build
-Write-Host "::group::Get Modified Files"
-try {
-    $modifiedFiles = @(Get-ModifiedFiles -baselineSHA $baselineWorkflowSHA)
-    OutputMessageAndArray -message "Modified files" -arrayOfStrings $modifiedFiles
-}
-catch {
-    Write-Host "Failed to calculate modified files, building all projects"
-    $buildAllProjects = $true
-    $modifiedFiles = @()
+    Write-Host "::group::Get Modified Files"
+    try {
+        $buildAllProjects, $modifiedFiles = Get-ModifiedFiles -baselineSHA $baselineWorkflowSHA
+        OutputMessageAndArray -message "Modified files" -arrayOfStrings $modifiedFiles
+    }
+    catch {
+        OutputNotice -message "Failed to calculate modified files since $baselineWorkflowSHA, building all projects"
```

Comment:
Can we write the actual error message in the log as well? Would make it easier to debug why we couldn't calculate modified files

Reply:
Also, I still believe a warning is better suited here.

Reply:
> Also, I still believe a warning is better suited here.

Agree 👍